src/components/DocumentInfo/DocumentInfo.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import { LabelBordered } from "./../Helpers/LabelBordered";
+import { AddressMultiple } from "./AddressMultiple";
+
+export const DocumentInfo = () => {
+  const registryAddress = "0x37242939c5B691D0A9402b21cbD61aCd287e552b";
+  const adminAddress = "0x1D350495B4C2a51fBf1c9DEDadEAb288250C703e";
+  const holderAddress = "0x28F7aB32C521D13F2E6980d072Ca7CA493020145"; // 0x28F7aB32C521D13F2E6980d072Ca7CA493020145
```

Comment:
I'm curious about the comment ? same below ? looks the same as the content

Reply:
i can remove, it was for me to revert the addresses when i copy paste the same adminAddr into holderAddr or BeneAddr to reflect the UI change

Reply:
ok :) just wondering, I think it's ok can remove when it's not static anymore